tracing/tracing.go

```diff
@@ -34,36 +28,11 @@ func ContextWithTracer(ctx context.Context, tracer opentracing.Tracer) context.C
 	return context.WithValue(ctx, tracerKey, tracer)
 }
 
-// tracerFromContext extracts opentracing.Tracer from the given context.
-func tracerFromContext(ctx context.Context) opentracing.Tracer {
+// TracerFromContext extracts opentracing.Tracer from the given context.
+func TracerFromContext(ctx context.Context) opentracing.Tracer {
```

Comment:
Do we need this? Can we move it to opentracing, so "common" tracing pkg is vendor free?